sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -2492,51 +2492,4 @@ class SQLQuerySuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
       }
     }
   }
-
-  test("SPARK-26560 Spark should be able to run Hive UDF using jar regardless of " +
```

Comment:
This test is moved to `HiveUDFDynamicLoadSuite` - now it's being tested with 5 available Hive UDF types.